ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-file-answer/activityFileAnswer.component.html

```diff
@@ -26,35 +27,37 @@
     </div>
 
     <div class="uploaded-file">
-        <ng-container *ngIf="fileNameToUpload">
+        <ng-container *ngIf="isLoading">
             <button mat-icon-button color="primary"
                     class="cancel-upload-btn"
                     [attr.aria-label]="'SDK.FileUpload.CancelUpload' | translate"
                     (click)="cancelUpload()"
                     #cancelUploadBtn>
                 <mat-icon>cancel</mat-icon>
             </button>
-            <div class="title">{{'SDK.FileUpload.Uploading' | translate: {filename: fileNameToUpload} }}</div>
+            <div class="title">{{'SDK.FileUpload.Uploading' | translate }}</div>
             <ddp-loading [loaded]="!isLoading" class="loading-bar"></ddp-loading>
         </ng-container>
 
-        <ddp-activity-file-answer-success *ngIf="uploadedFile"></ddp-activity-file-answer-success>
+        <ddp-activity-file-answer-success *ngIf="uploadedFiles && uploadedFiles.length > 0"></ddp-activity-file-answer-success>
```

Comment:
_**#minor**_

`uploadedFiles && uploadedFiles.length > 0` can be shortened like `uploadedFiles?.length`

Reply:
Sure, thank you for your feedback. Done!